src/test/java/org/kcctl/command/DeleteOffsetsCommandTest.java

```diff
@@ -0,0 +1,148 @@
+import static org.awaitility.Awaitility.await;
+
+@QuarkusTest
+@TestProfile(IntegrationTestProfile.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+@SkipIfConnectVersionIsOlderThan("3.6")
+class DeleteOffsetsCommandTest extends IntegrationTest {
+
+    private static final long OFFSET_AVAILABILITY_TIMEOUT_SECONDS = 30;
+
+
+    private final ObjectMapper mapper = new ObjectMapper();
+
+    @InjectCommandContext
+    KcctlCommandContext<DeleteOffsetsCommand> deleteContext;
+
+    @InjectCommandContext
+    KcctlCommandContext<GetOffsetsCommand> getContext;
+
+    @InjectCommandContext
+    KcctlCommandContext<StopConnectorCommand> stopContext;
+
+    @Test
+    public void delete_offsets_of_running_connector_should_return_error() {
+
+        registerTestConnector("delete-offsets-test1");
+
+        await()
+                .atMost(Duration.ofSeconds(OFFSET_AVAILABILITY_TIMEOUT_SECONDS))
+                .until(() -> {
+                    deleteContext.reset();
+
+                    if (!isOffsetAvailable("delete-offsets-test1")) {
+                        // No offsets have been committed yet; wait a little longer for an offset commit to complete
+                        return false;
+                    }
+
+                    int exitCode = deleteContext.commandLine().execute("delete-offsets-test1");
+                    String errorMessage = deleteContext.error().toString();
+
+                    assertThat(exitCode).isEqualTo(CommandLine.ExitCode.SOFTWARE);
+                    assertThat(errorMessage).contains("Connectors must be in the STOPPED state before their offsets can be modified. This can be done for the specified connector by issuing a 'PUT' request to the '/connectors/delete-offsets-test1/stop' endpoint" );
+                    return true;
+                });
+    }
+
+    @Test
+    public void offsets_are_correctly_deleted() {
+
+        registerTestConnector("delete-offsets-test1");
+
+        await()
+                .atMost(Duration.ofSeconds(OFFSET_AVAILABILITY_TIMEOUT_SECONDS))
+                .until(() -> {
+                    deleteContext.reset();
+
+                    if (!isOffsetAvailable("delete-offsets-test1")) {
+                        // No offsets have been committed yet; wait a little longer for an offset commit to complete
+                        return false;
+                    }
+
+                    stopContext.runAndEnsureExitCodeOk("delete-offsets-test1");
+
+                    int exitCode = deleteContext.commandLine().execute("delete-offsets-test1");
+                    String output = deleteContext.output().toString();
+                    AlterResetOffsetsResponse response = mapper.readValue(output, AlterResetOffsetsResponse.class);
+
+                    assertThat(exitCode).isEqualTo(CommandLine.ExitCode.OK);
+                    assertThat(response.message()).isEqualTo("The offsets for this connector have been reset successfully");
```

Comment:
This also doesn't seem to belong in the `await` block. I think the flow we need is:

1. Wait for one or more offsets to be committed by the connector
2. Stop the connector
3. Wait for the connector to be stopped (i.e., no tasks left and the `Connector` is in the `STOPPED` state)
4. Delete offsets, which should succeed immediately by this point